tests/projects/test_canvas_snapshotter.py

```
@@ -68,3 +68,34 @@ async def test_export_now_synchronous(env):
+
+
+@pytest.mark.asyncio
+async def test_tldr_render_writes_off_the_event_loop(env, monkeypatch):
+    """The .tldr snapshot write must not stall the loop.
+
+    ``atomic_write_text`` fsyncs the file and the parent directory; a board
+    snapshot is the largest of these writes, so it is the worst offender.
+    """
+    import tinyagentos.projects.canvas.snapshotter as sn
+
+    ps, cs, snap, p, data_root = env
```

Ruff (0.16.3)

[warning] Unpacked variable `ps` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)